Contribution-and-Creation/README.md

```diff
@@ -0,0 +1,47 @@
+# OSPO 101 Training Extension Module 6 & 7
+## Open Source Contribution and Creation
+
+This course series is an extension of modules 6 and 7 from the OSPO 101 training course and is based on the [content from the outbound open source guide](https://github.com/todogroup/outbound-oss). Before taking this course, we encourage students to go through the introductory courses on using and creating open source projects:
+
+* [OSPO 101 - Module 6: Understanding Upstream Open Source Projects](https://github.com/todogroup/ospo-career-path/blob/main/OSPO-101/module6/README.md)
+* [OSPO 101 - Module 7: Open Source Creation](https://github.com/todogroup/ospo-career-path/tree/main/OSPO-101/module7)
+
+***
+
+### Program Overview
+
+For people at organizations looking to enhance their open source and/or IT strategy (people shaping the process), delving into outbound open source – which involves contributing to existing projects or initiating new ones – can be a transformative experience. 
+This training course is designed to guide open source managers adn stakeholders within organizations through this journey.
+
+This program is designed around three main thematic areas, which will be formatted into modules:
+
+**MODULE 1 - COLLABORATION**
+
+* Transitioning from usage (also called consumption) to Contribution
+  * Diving Deep into starting your organization's journey in contributing to open source projects
+  * Exploring various contribution models and procedures suitable for enterprises of all scales
+  * Managing and encouraging spare-time contributions
+    
+**MODULE 2 - CREATION**
+
+* Embarking on the organization's open source project releases
+  * An in-depth look at the  lifecycle of open source projects.
+  * Understanding the strategic advantages of launching open source initiatives for your organization.
+  * Navigating through legal intricacies, including Intellectual Property (IP) and licensing considerations.
+  * Mastering administrative and technical aspects, from repository management to selecting the right tools for your project.
+  * 
+**MODULE 3 - SUSTAINABILITY AND RESILIENCE**
+
+* Growing and nurturing and keeping an open source project resilient
+   * Techniques and best practices for promoting the open source project that matters to your organization
+   * Building and managing vibrant communities around the open soruce projects that matter to your organization.
+   * An introduction to open source software analytics and crucial open source metrics to gauge success and areas of improvement
+
+By the end of this training course, participants will be well-equipped with the knowledge and tools necessary to contribute to open source projects and kickstart and release open source projects, ensuring they align with their organization's strategic objectives and foster a culture of collaboration and innovation.
```

Comment:
This sounds like it's targeted at people who are actively doing contributions themselves. This is a bit in conflict with the definition of the audience in the next section. It should be made clear that the training addresses open source managers not open source developers, and that it's about shaping the contribution process.